docs/static/releasenotes.asciidoc

```diff
@@ -1,16 +1,390 @@
 [[releasenotes]]
 == Release Notes
 
-coming[8.0.0]
-
 This section summarizes the changes in the following releases:
 
+* <<logstash-8-0-0,Logstash 8.0.0>>
 * <<logstash-8-0-0-rc2,Logstash 8.0.0-rc2>>
 * <<logstash-8-0-0-rc1,Logstash 8.0.0-rc1>>
 * <<logstash-8-0-0-beta1,Logstash 8.0.0-beta1>>
 * <<logstash-8-0-0-alpha2,Logstash 8.0.0-alpha2>>
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
+[[logstash-8-0-0]]
+=== Logstash 8.0.0 Release Notes
+
+The list combines release notes from the 8.0.0-alpha1, -alpha2, -beta1, -rc1 and -rc2 releases.
+
+==== Logstash core
```

Comment:
Instead of "Logstash Core," let's continue the headings with anchor approach we started in 7.x branches, and group items into appropriate categories. 

So something like:

```
[[breaking-8.0.0]]
==== Breaking changes

   <At the end of the list, please include a link to the breaking changes section (`<<breaking-8.0>>`) for a complete list of all breaking changes across 8.0>

[[features-8.0.0]]
==== New features and enhancements

[[notable-8.0.0]]
==== Performance improvements and notable issues fixed
```

Reply:
The `[[breaking-8.0.0]]` anchor in release notes and the `[[breaking-8.0]]` anchor for the Breaking Changes section should be unique and so they shouldn't cause problems.  (Re-used anchors crash the docs  build.)